style.css

```diff
@@ -1,66 +1,151 @@
-html{
-    font-family: 'Josefin Sans', sans-serif;
-    box-sizing: border-box;
-  }
+html {
+  font-family: 'Josefin Sans', sans-serif;
+}
+
+#home {
+  background-image: url("https://images.pexels.com/photos/434337/pexels-photo-434337.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+  background-size: 100%;
+  height: 500px;
+  padding-top: 40px;
+  padding-left: 10px;
+}
+
+.welcome {
+  font-size: 45px;
+  color:#38ada9;
+  text-align: left;
+  margin-top: 20%;
+  margin-left: 25%;
+  margin-right: 25%;
+}
+
+.welcome_subtitle {
+  font-size: 30px;
+  color: #b2bec3;
```

Comment:
lets use custom properties for color usage 

https://dockyard.com/blog/2018/06/05/new-css-features-on-the-component-level-part-2-of-2-responsive-css-variables

https://dockyard.com/blog/2018/10/24/how-to-use-css-variables-for-svg-gradients